src/Model/Subsite.php

```diff
@@ -932,13 +931,24 @@ public function getMembersByPermission($permissionCodes = ['ADMIN'])
 
         $SQL_permissionCodes = join("','", $SQL_permissionCodes);
 
+        $schema = DataObject::getSchema();
+        $groupTable = $schema->tableName(Group::class);
+        $groupMembersTable = "{$groupTable}_{Members}";
+
         return DataObject::get(
             Member::class,
-            "\"Group\".\"SubsiteID\" = $this->ID AND \"Permission\".\"Code\" IN ('$SQL_permissionCodes')",
-            '',
-            'LEFT JOIN "Group_Members" ON "Member"."ID" = "Group_Members"."MemberID"
-            LEFT JOIN "Group" ON "Group"."ID" = "Group_Members"."GroupID"
-            LEFT JOIN "Permission" ON "Permission"."GroupID" = "Group"."ID"'
+            $schema->sqlColumnForField(Group::class, 'SubsiteID') . " = $this->ID AND "
+                . $schema->sqlColumnForField(Permission::class, 'Code')
+                . " IN ('$SQL_permissionCodes')"
+        )->leftJoin(
+            $groupMembersTable,
+            $schema->sqlColumnForField(Member::class, 'ID') . " = \"{$groupMembersTable}\".\"MemberID\""
+        )->leftJoin(
+            $groupTable,
+            $schema->sqlColumnForField(Group::class, 'ID') . " = \"{$groupMembersTable}\".\"GroupID\""
+        )->leftJoin(
+            $schema->tableName(Permission::class),
+            $schema->sqlColumnForField(Permission::class, 'ID') . ' = ' . $schema->sqlColumnForField(Group::class, 'ID')
```

Comment:
Since I had to refactor this anyway, I decided to remove the hardcoded references since those weren't following best practices.
The query should ultimately be identical.